test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -99,7 +94,10 @@ module.exports = params => {
 								return done(err);
 							}
 
-							if ((numOfConnections - 20) <= expectedConnectionsAfterBlacklisting) {
+							if (
```

Comment:
Same here - why not expect?

Reply:
I've taken the same way @ManuGowda has used in another test. The result is the same.